Resources/views/Collector/db.html.twig

```diff
@@ -254,6 +254,11 @@
                                         &nbsp;&nbsp;
                                         <a class="link-inverse" href="{{ path('_profiler', { panel: 'db', token: token, page: 'explain', connection: connection, query: i }) }}" onclick="return explain(this);" data-target-id="explain-{{ i }}-{{ loop.parent.loop.index }}">Explain query</a>
                                     {% endif %}
+
+                                    {% if query.backtrace is defined %}
+                                        &nbsp;&nbsp;
+                                        <a href="#" class="sf-toggle link-inverse" data-toggle-selector="#backtrace-query-{{ i }}-{{ loop.parent.loop.index }}" data-toggle-alt-content="Hide query backtrace">View query backtrace</a>
```

Comment:
Maybe make things easier for the reader:
```suggestion
                                        <a 
                                            href="#" 
                                            class="sf-toggle link-inverse" 
                                            data-toggle-selector="#backtrace-query-{{ i }}-{{ loop.parent.loop.index }}" 
                                            data-toggle-alt-content="Hide query backtrace"
                                        >View query backtrace</a>
```

Reply:
I prefer to keep inline version, like other links in this template